env.default

```diff
@@ -34,7 +34,7 @@ XSS_PROTECTION=True
 # CSP config
 
 CSP_DEFAULT_SRC="'none'"
-CSP_SCRIPT_SRC='self' 'unsafe-inline' cdn.optimizely.com https://www.google-analytics.com/analytics.js https://c.shpg.org/352/sp.js
+CSP_SCRIPT_SRC='self' 'unsafe-eval' 'unsafe-inline' cdn.optimizely.com https://www.google-analytics.com/analytics.js
```

Comment:
if anyone knows what `https://c.shpg.org/352/sp.js` is, do let me know (we should probably have that explained in dev docs somewhere because just looking at the url's content isn't super indicative that this is a safe script at all)

Reply:
As secondary comment: it looks like unsafe-eval is necessary for the a jquery-ui datepicker. If we can fix that dependency we can hopefully get rid of `unsafe-eval` again. I'm not super keen on landing this PR until that's sorted.

Reply:
It's share progress. @ScottDowne Just put it in.